### DISCUSSION TITLE
Adding Elestio as One click deployment option

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [towfiqi/serpbear#125] [Ideas]
Hello there,
I am Kaiwalya from Elestio. We have been providing an option of hosting Serpbear as a fully managed application on Elestio as [shown here](https://elest.io/open-source/serpbear). I think it would be beneficial if we can add it here in the documentation. 
Additionally, we also share our revenue with the projects we support. I couldn't find the docs to create a Pull request. It would be helpful if you can redirect me to the docs repo/directory. 
![image](https://github.com/towfiqi/serpbear/assets/53310847/b7d2b94d-4b2b-4ab6-97cc-b35ece4de658)

**Comment by towfiqi** [comment]
Can you please write a short documentation like this one here: https://docs.serpbear.com/deployment/deploying-to-pikapods-1-click and share the documentation here?

**Comment by kaiwalyakoparkar** [comment]
@towfiqi Certainly, can you please guide me on where I can find the directory containing the documentation? I am unable to find it

**Reply by towfiqi** [reply]
The docs are not hosted in GitHub.

**Reply by kaiwalyakoparkar** [reply]
Oh, where should I write the documentation? Should I write the md file and share with you here?

**Reply by towfiqi** [reply]
Yes, you can just paste the markdown content in the comment box of this page. 
Once you post the comment, I can click the edit button, and copy the md content of your comment.

**Comment by kaiwalyakoparkar** [comment]
```md
[Elestio](https://elest.io/) covers everything from deployment to security, monitoring, and expert support in a few clicks. You get full root SSH access to Serpbear instance, advanced customizations, and a web dashboard for stack and env var changes.
New users receive a $20 welcome credit

Step 1:
[Create an account](https://dash.elest.io/) on Elestio, add your payment details and get started

Step 2:
After logging in, you can launch SerpBear using the link below:
[![Deploy](https://pub-da36157c854648669813f3f76c526c2b.r2.dev/deploy-on-elestio-black.png)](https://elest.io/open-source/serpbear)

Step 3:
Select service provider, regions, support tier, and setup env vars and you are done! You have successfully deployed Serpbear on Elestio started.
```
@towfiqi Let me know if you need any changes in the above document

**Comment by towfiqi** [comment]
Added to docs: https://docs.serpbear.com/deployment/deploying-to-elestio-1-click

**Comment by kaiwalyakoparkar** [comment]
Hey @towfiqi, hope you are doing well. We just revamped our "Deploy on Elestio" button. It would be awesome if you can update it in the docs 🙌 

[![Deploy on Elestio](https://elest.io/images/logos/deploy-to-elestio-btn.png)](https://elest.io/open-source/serpbear)

```markdown
[![Deploy on Elestio](https://elest.io/images/logos/deploy-to-elestio-btn.png)](https://elest.io/open-source/serpbear)
```
